README.md

```diff
@@ -31,7 +31,7 @@ This plugin is compatible with CSI versions [v1.0.0](https://github.com/containe
 | v0.2.0 (alpha)                       | no            | yes  | no   | no   |
 | v0.3.0 (beta)                        | no            | no   | yes  | yes  |
 | v0.4.0 (beta)                        | no            | no   | yes  | yes  |
-| v0.5.0 (beta)                        | no            | no   | no   | yes  |
```

Comment:
should we change everything to just have the vX.Y?

Reply:
yeah makes sense to me. Fixed.